src/main/drivers/resource.h

```diff
@@ -119,11 +120,18 @@ typedef enum {
 
 typedef struct resourceOwner_s {
     resourceOwner_e owner;
-    uint8_t resourceIndex;
+    uint8_t index;
 } resourceOwner_t;
 
-extern const char * const ownerNames[OWNER_TOTAL_COUNT];
+// Sentinels with global address identity; compare by pointer
```

Comment:
Or simply test `->owner` instead.
But singletons are still useful

Reply:
I will leave the singletons... for now.